### DISCUSSION TITLE
UPDATED BM1000-HDMI (HDMI Encoder) Dolby 2.0

### DISCUSSION THREAD

**Discussion by mackandall** [OpenSageTV/Community#52] [Capture Devices]
Just a heads up. The newer versions of this capture device appear to now be able to do ac3. It comes through as Dolby 2.0. I have been using it in both sagetv and nextpvr and it sound good. It is louder and more detailed than the AAC, at least to me. 
[
](https://www.amazon.com/Encoder-Broadcast-Protocols-Facebook-Platforms/dp/B074H1X7QJ/ref=sr_1_1_sspa?crid=33IGSGS04LVEB&keywords=4k+iptv+encoder+box&qid=1690079348&sprefix=%2Caps%2C98&sr=8-1-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&psc=1)
[
](https://www.amazon.com/Encoder-Broadcast-Protocols-Facebook-Platforms/dp/B07MM9HGLZ/ref=sr_1_2_sspa?crid=33IGSGS04LVEB&keywords=4k+iptv+encoder+box&qid=1690079348&sprefix=%2Caps%2C98&sr=8-2-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&psc=1)

**Comment by JustFred51** [comment]
Can you provide a link to the device you bought?

**Reply by mackandall** [reply]
Sorry, just seeing this...

There is also a single encoder version.  

https://www.amazon.com/URayCoder-Cost-Effective-Streaming-Broadcast-Transmitter/dp/B07TKMPCZH/ref=sr_1_3?crid=30F4RROLS0L1O&keywords=uray+encoder+4+port&qid=1690770039&sprefix=uray+4+%2Caps%2C279&sr=8-3

**Comment by eelpout** [comment]
> Just a heads up. The newer versions of this capture device appear to now be able to do ac3. It comes through as Dolby 2.0. I have been using it in both sagetv and nextpvr and it sound good. It is louder and more detailed than the AAC, at least to me. [ ](https://www.amazon.com/Encoder-Broadcast-Protocols-Facebook-Platforms/dp/B074H1X7QJ/ref=sr_1_1_sspa?crid=33IGSGS04LVEB&keywords=4k+iptv+encoder+box&qid=1690079348&sprefix=%2Caps%2C98&sr=8-1-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&psc=1) [ ](https://www.amazon.com/Encoder-Broadcast-Protocols-Facebook-Platforms/dp/B07MM9HGLZ/ref=sr_1_2_sspa?crid=33IGSGS04LVEB&keywords=4k+iptv+encoder+box&qid=1690079348&sprefix=%2Caps%2C98&sr=8-2-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&psc=1)

you mean it has analog matrix encoding (as opposed to discrete)? haven't these always done that? I don't have one but I assume it would do the same thing as my HD-PVR's do over the analog RCA audio outputs.

I gave up on my HD-PVR's and Dolby 5.1 from a DirecTV optical source years ago. Too many glitches of either out of sync problems or no audio tracks altogether. I just set the DirecTV's output with 5.1 disabled.

FYI. [this is the link](https://www.amazon.com/Encoder-Broadcast-Protocols-Facebook-Platforms/dp/B07MM9HGLZ/ref=sr_1_2_sspa?crid=33IGSGS04LVEB&keywords=4k+iptv+encoder+box&qid=1690079348&sprefix=%2Caps%2C98&sr=8-2-spons&sp_csd=d2lkZ2V0TmFtZT1zcF9hdGY&psc=1) to the device. It was there in the post, just hidden.

**Comment by mackandall** [comment]
Just coming over from the old forums. Didnt realize the links were hidden. 

"you mean it has analog matrix encoding (as opposed to discrete)? haven't these always done that?"

The earlier versions of the device did not encode audio as ac3 as far as I can tell. I have two versions, both uray branded. The older one is a single channel 1080p and the other a 4 channel version that does 4k. The latter, which is newer, has ac3 listed as an option for the audio encoding instead of only aac, aac+, mp3 and mp2. 

That ac3 dual channel is read by sagetv as dolby 2.0. That seems to be what the software on the devices limited to for whatever reason. As far as 5.1  goes the hdpvr2 is the only solid performer I have found. It is not perfect, but you do get discrete channels as far as my untrained ears can tell. For some reason though, the ac3 2.0 on the newer uray device sounds richer to me.

**Reply by PhilipCRoberts** [reply]
I can't get the old forums to load whatsoever.  This device looks interesting.  Am I correct in my understanding that it can import web streams?  Livestream maybe?

**Comment by eelpout** [comment]
_> I can't get the old forums to load whatsoever._

the "php" hyptertext preprocessor no longer works on the old site, only static pages are there. so if you are relying on say Google searches, which point to the ".php" entries, you'll have to massage them. in addition, you'll need to mark the site as "safe" in your browser.

Google result for "BM1000-HDMI" and forums.sagetv.com:

_https://forums.sagetv.com/forums/showthread.php?t=63177_  (gives 404 error)

change to:

_https://forums.sagetv.com/forums/showthread_t_63177.html_ (BM1000-HDMI (HDMI Encoder) Review page)

this also means the old website search function no longer works either, sadly.

**Reply by PhilipCRoberts** [reply]
Got it, thanks!

**Comment by TGsell1** [comment]
I use two of these and they work really well w Sage via OpenDCT , CableCard Windows Media Center setup. As long as  CableCard is still viable I’m good.  Sent from my iPhoneOn Jul 30, 2023, at 10:22 PM, boukmandutty ***@***.***> wrote:﻿
Sorry, just seeing this...
There is also a single encoder version.
https://www.amazon.com/URayCoder-Cost-Effective-Streaming-Broadcast-Transmitter/dp/B07TKMPCZH/ref=sr_1_3?crid=30F4RROLS0L1O&keywords=uray+encoder+4+port&qid=1690770039&sprefix=uray+4+%2Caps%2C279&sr=8-3

**Comment by PhilipCRoberts** [comment]
So this can be used to import web streams?

**Comment by TGsell1** [comment]
Yes.  I have used it to manually import streams from a miniPC via HDMI.  I
didn’t try to work out a script to do this manually however I did for 2
Windows Media Center and CableCard machines and it works flawlessly.
Anyway the URay is very useful.
